pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -718,23 +380,106 @@ PageServer.prototype = {
+        return (
+            <Card>
+                <CardHeader>{_("Configuration")}</CardHeader>
+                <CardBody>
+                    <table className="pf-c-table pf-m-grid-md pf-m-compact">
+                        <tbody>
+                            <tr>
+                                <th>{_("Hostname")}</th>
+                                <td>
+                                    <div role='group'>
+                                        {this.props.hostname}
+                                        <Button variant='link' isInline onClick={() => $('#system_information_change_hostname').modal('show')}>{_("edit")}</Button>
+                                    </div>
+                                </td>
+                            </tr>
+
+                            <tr>
+                                <th>{_("System time")}</th>
+                                <td>
+                                    <div role="group">
+                                        <span id="systime-tooltip">
+                                            <a role="button" tabIndex="0" className="systime-privileged"
+                                                id="system_information_systime_button"
+                                                onClick={() => this.change_systime_dialog.display(this.server_time)}>
+                                                {this.state.serverTime}
+                                            </a>
+                                        </span>
+                                        <a tabIndex="0" hidden id="system_information_systime_ntp_status"
+                                            role="button" data-toggle="tooltip"
+                                            data-placement="bottom" data-html="true" dangerouslySetInnerHTML={this.state.ntp_status_icon} />
+                                    </div>
+                                </td>
+                            </tr>
+
+                            <tr>
+                                <th>{_("Domain")}</th>
+                                <td><p id="system-info-domain" /></td>
+                            </tr>
+
+                            <tr>
+                                <th>{_("Performance profile")}</th>
+                                <td><span id="system-info-performance" /></td>
+                            </tr>
+
+                            <tr>
+                                <th>{_("Secure Shell keys")}</th>
+                                <td>
+                                    <a id="system-ssh-keys-link" data-toggle="modal" onClick={this.host_keys_show}
+                                        data-target="#system_information_ssh_keys">{_("Show fingerprints")}</a>
+                                </td>
+                            </tr>
+
+                            {this.state.sysroot_version_link_visible &&
+                            <tr>
+                                <th>{_("Version")}</th>
```

Comment:
If you want to keep this, please rename it to "OSTree version". (And, as already discussed, move to the System info card)